go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/emirpasic/gods v1.18.1
 	github.com/ethantkoenig/rupture v1.0.1
 	github.com/felixge/fgprof v0.9.2
+	github.com/fsnotify/fsnotify v1.5.4
```

Comment:
Do we want to continue using fsnotify? https://github.com/fsnotify/fsnotify/pull/459

Reply:
Well it is a dependency of unrolled which is why I used it in the first place. It also was only archived 4 days ago - almost 16 days after I started this work.

Reply:
https://github.com/fsnotify/fsnotify/issues/413 indicates why it was archived. I think there is a high chance of this being unarchived in the near future.

Reply:
I've switched to use https://github.com/syncthing/notify

Reply:
Fine either way. Shouldn't be a big problem for now(hope so)

Reply:
I've had to switch back as the notify framework opens up goroutines etc even if it isn't used.

Reply:
I just saw this linked in the issue, but fsnotify is now unarchived and maintained again, FYI.